core/src/main/java/com/radixdlt/RunningRadixNode.java

```diff
@@ -156,28 +150,9 @@ public void reportSelfStartupTime(Duration startupTimeMs) {
     this.injector.getInstance(Metrics.class).misc().nodeStartup().observe(startupTimeMs);
   }
 
-  public void installShutdownHook() {
-    Thread hook = new Thread(this::shutdownFromHook);
-    final var set = this.shutdownHook.compareAndSet(null, hook);
-    Preconditions.checkState(set, "already installed before");
-    Runtime.getRuntime().addShutdownHook(hook);
-  }
-
-  private void shutdownFromModule(String reason) {
-    final @Nullable Thread hook = this.shutdownHook.getAndSet(null);
-    if (hook != null) {
-      Runtime.getRuntime().removeShutdownHook(hook);
-    }
-    doShutdown(reason);
-  }
-
-  private void shutdownFromHook() {
-    doShutdown("a requested shutdown (via the Java shutdown hook)");
-  }
-
-  private void doShutdown(String reason) {
+  public void shutdown() {
```

Comment:
Now that this is only called by the shutdown hook, I have a slight preference to call this `onShutdown` or stick a JavaDoc to explain this is registered as the shutdown hook